src/mono/msbuild/android/build/AndroidBuild.targets

```diff
@@ -112,12 +112,20 @@
       <_AotOutputType>ObjectFile</_AotOutputType>
     </PropertyGroup>
 
-    <ItemGroup>
-      <MonoAOTCompilerDefaultAotArguments Condition="'$(TargetArchitecture)' == 'arm'" Include="mtriple=armv7-linux-gnueabi" />
-      <MonoAOTCompilerDefaultAotArguments Condition="'$(TargetArchitecture)' == 'arm64'" Include="mtriple=aarch64-linux-android" />
-      <MonoAOTCompilerDefaultAotArguments Condition="'$(TargetArchitecture)' == 'x86'" Include="mtriple=i686-linux-android" />
-      <MonoAOTCompilerDefaultAotArguments Condition="'$(TargetArchitecture)' == 'x64'" Include="mtriple=x86_64-linux-android" />
+    <PropertyGroup>
+      <_Triple Condition="'$(TargetArchitecture)' == 'arm'">armv7-linux-gnueabi</_Triple>
+      <_Triple Condition="'$(TargetArchitecture)' == 'arm64'">aarch64-linux-android</_Triple>
+      <_Triple Condition="'$(TargetArchitecture)' == 'x86'">i686-linux-android</_Triple>
+      <_Triple Condition="'$(TargetArchitecture)' == 'x64'">x86_64-linux-android</_Triple>
+    </PropertyGroup>
+
+    <PropertyGroup>
+      <_LdName>clang</_LdName>
+      <_LdOptions>-fuse-ld=lld</_LdOptions>
+      <_AsName>clang</_AsName>
+    </PropertyGroup>
```

Comment:
Can we extract these settings to a common .props file so that we don't specify them in the `AndroidSampleApp` again?

Reply:
https://github.com/dotnet/runtime/issues/110811